demo/project_demo04-SecOps/run.py

```diff
@@ -15,7 +15,7 @@
 
 
 def prepare():
-    cmd = ["cargo", "run", "--release"]
+    cmd = ["cargo", "run", "--release", "--", "10000"]
```

Comment:
I know this change is meant to make the demo go fast, but if we don't generate enough data to run the demo for at least half a minute, we don't really have a demo.